### PR TITLE
22w11a misc stuff

### PR DESCRIPTION
**Pull request by apple502j**
The final PR for 22w11a (hopefully).

### CODE REVIEW

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 1]
```diff
@@ -835,6 +835,24 @@ CLASS net/minecraft/class_2540 net/minecraft/network/PacketByteBuf
 	METHOD method_37453 getMaxValidator (Ljava/util/function/IntFunction;I)Ljava/util/function/IntFunction;
 		ARG 0 applier
 		ARG 1 max
+	METHOD method_42064 readRegistryValue (Lnet/minecraft/class_2359;)Ljava/lang/Object;
```
Comment (liach) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:841]: Mind update that giant table in the class javadoc as well?

Also would be nice to note that callers are to ensure the registries/indexed iterables are matched on two sides before using these

File: mappings/net/minecraft/item/ItemStack.mapping [hunk 2]
```diff
@@ -171,6 +174,11 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_33262 onItemEntityDestroyed (Lnet/minecraft/class_1542;)V
 		ARG 1 entity
 	METHOD method_40133 streamTags ()Ljava/util/stream/Stream;
+	METHOD method_41406 itemMatches (Lnet/minecraft/class_6880;)Z
+		ARG 1 itemEntry
+	METHOD method_41407 itemMatches (Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_41409 getRegistryEntry ()Lnet/minecraft/class_6880;
```
Comment (liach) [inline mappings/net/minecraft/item/ItemStack.mapping:181]: In `BlockState` we have `getBlockRegistryEntry`. Should this just be `getRegistryEntry` or `getItemRegistryEntry` for consistency?

Comment (apple502j) [reply mappings/net/minecraft/item/ItemStack.mapping:181]: @liach I think you have the outdated version of Shnupbups' branch? That was renamed before the merge.

File: mappings/net/minecraft/network/PacketByteBuf.mapping (hunk 1 above)
Comment (apple502j) [reply mappings/net/minecraft/network/PacketByteBuf.mapping:841]: Ah yeah, fixing

**Pull request merged**